### DISCUSSION TITLE
Concerning amount of spam in this Discussion forum

### DISCUSSION THREAD

**Discussion by faridnec** [community/community#105809] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This discussion forum is filled with lots of spam on a level that is concerning. I hope the admin can somehow manage to filter the discussion so it will give opportunity for those in need to ask things or deliver feedback

**Comment by sapalazz** [comment]
I agree, it's getting out of hand. I was thinking it was time we flagged it. 👍

**Comment by itzzjb** [comment]
Yes, It's getting very annoying.
